lib/osf-components/addon/components/cedar-metadata-renderer/template.hbs

```diff
@@ -16,13 +16,43 @@
                     {{/if}}
                 </div>
                 <div local-class='button-container'>
+                    <OsfLink
+                        data-test-download-icon
+                        local-class='spacer'
+                        aria-label={{t 'metadata.download'}}
+                        @href={{@cedarMetadataRecord.links.metadataDownload}}
+                        @target='_blank'
+                    >
+                        <FaIcon @icon='download' />
+                        <EmberTooltip
+                            @side='right'
+                        >
+                            {{t 'metadata.download'}}
+                        </EmberTooltip>
+                    </OsfLink>
+                    {{#unless @displayFileMetadata}}
+                        <OsfLink
+                            local-class='spacer'
+                            data-test-download-button
+                            aria-label={{t 'metadata.download'}}
+                            @href={{@cedarMetadataRecord.links.metadataDownload}}
+                            @target='_blank'
+                        >
+                            {{t 'general.download'}}
+                        </OsfLink>
+                    {{/unless}}
```

Comment:
So does this mean that there will be two links to download on the file view? Could you do something like this to just use one link?
```suggestion
                    <OsfLink
                        data-test-download-icon
                        local-class='spacer'
                        aria-label={{t 'metadata.download'}}
                        @href={{@cedarMetadataRecord.links.metadataDownload}}
                        @target='_blank'
                    >
                        <FaIcon @icon='download' />
                        {{#if @displayFileMetadata}}
                            <span>
                                local-class='spacer'
                                data-test-download-text
                            >
                                {{t 'general.download'}}
                            </span>
                        {{else}}
                            <EmberTooltip
                                @side='right'
                            >
                                {{t 'metadata.download'}}
                            </EmberTooltip>
                        {{/if}}
                    </OsfLink>
```

I think this will work with the EmberTooltip as well, but may need a little bit of tweaking